language/stdlib/modules/libra_coin.move

```diff
@@ -85,6 +123,14 @@ module LibraCoin {
         T{ value: amount }
     }
 
+    // Return a coin new coin whose value is coin_ref.value
```

Comment:
Typo: should be "...a new coin...", I think.

Reply:
Agreed, but I actually don't end up needing this procedure in the revised module.